cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -153,6 +171,12 @@ public static EntityTopicOperator fromCrd(Reconciliation reconciliation,
             if (kafkaAssembly.getSpec().getEntityOperator().getTemplate() != null)  {
                 result.templateRoleBinding = kafkaAssembly.getSpec().getEntityOperator().getTemplate().getTopicOperatorRoleBinding();
             }
+            
+            result.cruiseControlEnabled = kafkaAssembly.getSpec().getCruiseControl() != null;
+            if (result.cruiseControlEnabled) {
+                result.rackAwarenessEnabled = kafkaAssembly.getSpec().getKafka().getReplicas() > 1
```

Comment:
This is a good point .... actually:
* We have a check that we don't deploy Cruise Control with a single Kafka node. So the check for number of nodes should be done already for the `cruiseControlEnabled` flag.
* `.spec.kafka.replicas` should not be used for a replica count as it might not be set anymore and might not have the number of nodes.

Reply:
Makes sense. So you say something like the following should be fine:

```sh
result.cruiseControlEnabled = kafkaAssembly.getSpec().getCruiseControl() != null;
result.rackAwarenessEnabled = result.cruiseControlEnabled && kafkaAssembly.getSpec().getKafka().getRack() != null;
```